src/helpers/current-page-helper.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Yoast\WP\SEO\Helpers;
 
+use WP_Post;
```

Comment:
Why is this needed?

Reply:
Leftover from autocomplete, has been removed.